packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -662,6 +717,56 @@ describe("wrapErrorAndLog", () => {
     assert(mockLogger.matchEvents([{
         eventName: "WrapError",
         wrappedByErrorInstanceId: newError.errorInstanceId,
+        errorInstanceId: newError.errorInstanceId,
         error: "hello",
      }]), "Expected the 'WrapError' event to be logged");
 });
+
+describe.only("Error Discovery", () => {
```

Comment:
Is this `only` intentional?

Reply:
You win a cookie for catching this!  Embarrassed I let this slip in.  Not my first time doing so...  :(